src/main/java/lottery/domain/LotteryRank.java

```diff
@@ -0,0 +1,35 @@
+package lottery.domain;
+
+import java.util.Arrays;
+
+public enum LotteryRank {
+    FIRST_PRIZE(6, 2_000_000_000),
+    SECOND_PRIZE(5, 1_500_000),
+    THIRD_PRIZE(4, 50_000),
+    FOURTH_PRIZE(3,5_000);
+
+    private final int matchNumberCounts;
+    private final long prizeMoney;
+
+    private LotteryRank(int matchNumberCounts, long prizeMoney) {
+        this.matchNumberCounts = matchNumberCounts;
+        this.prizeMoney = prizeMoney;
+    }
+
+    public static LotteryRank valueOf(int matchNumberCounts) {
+        return Arrays.stream(LotteryRank.values())
+                .filter(lotteryRank -> lotteryRank.getMatchNumberCounts() == matchNumberCounts)
+                .findAny()
+                .orElseThrow(() -> new IllegalArgumentException(ErrorMessages.INVALID_MATCH_NUMBER_COUNTS));
+    }
+
+    public double calculatePrizeMoneySum(int winnerTicketCounts) {
+        return (double)prizeMoney * winnerTicketCounts;
+    }
+
+    public long getPrizeMoney() { return prizeMoney; }
```

Comment:
```suggestion
    public long getPrizeMoney() { 
        return prizeMoney; 
    }
```
작성하신 컨벤션도 사용될 수 있지만, 프로젝트 내에서는 braces 컨벤션에 대한 규칙은 통일해주세요!